### PR TITLE
Add coverity support

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

**Comment by grahamwhaley** [comment]
lgtm

File: .travis.yml [hunk 1]
```diff
@@ -26,3 +32,12 @@ notifications:
...
     on_success: change
     on_failure: always
+
+addons:
+  coverity_scan:
+    project:
+      name: "01org/cc-oci-runtime"
+      description: "Build submitted via Travis CI"
+    build_command_prepend: "./autogen.sh --enable-cppcheck --enable-valgrind; make clean"
+    build_command: "make -j 4"
```
Comment (dlespiau) [inline .travis.yml]: That made me look at what travis provides:

https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments

2 cores, so maybe we can even push that to 5? Doing it for the main build looks useful as well.

File: .travis.yml [hunk 2]
```diff
@@ -26,3 +32,12 @@ notifications:
...
+    branch_pattern: coverity_scan
```
Comment (dlespiau) [inline .travis.yml]: So is the plan to push a coverity branch every day as part of the daily release process?

**Comment by dlespiau** [comment]
lgtm, maybe the make -j5 thing would be nice

File: .travis.yml (hunk 1 above)
Comment (jodh-intel) [inline .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (jodh-intel) [inline .travis.yml]: Yes - see https://github.com/01org/cc-oci-runtime/wiki/Release-Process.

**Pull request merged**